gspread/models.py

```diff
@@ -563,6 +564,36 @@ def insert_row(self, values, index=1):
                 newcell.value = rows_after_insert[r - 1][c - 1]
                 updated_cell_list.append(newcell)
         self.update_cells(updated_cell_list)
+    
+
+    def insert_column(self, values, index=1):
+        self.add_cols(1)
+        data_width = len(values)
+        if self.row_count < data_width:
+            self.resize(rows=data_width)
+	
+	all_cells = self.get_all_values()
+        j=0
```

Comment:
I guess this is not one of the recent commits. Anyways, thanks for checking this, I have re-checked and I don't see any indentation problems.